src/main/java/pl/javastart/task/GuessGame.java

```diff
@@ -0,0 +1,27 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class GuessGame {
+    Scanner scanner = new Scanner(System.in);
```

Comment:
Scanner może być zdefiniowany wewnątrz metody guessNumber(). Jeżeli coś jest używane tylko w jednej metodzie, to nie definiuj tego jako pole klasy